demos/ret2spec_cyclic.cc

```diff
@@ -95,6 +95,7 @@ static bool ReturnsTrue(int counter) {
        i += kCacheLineSize) {
     CLFlush(&stack_mark + i);
   }
+  CLFlush(stack_mark_pointers.back());
```

Comment:
Useful to put this in a utility function, something like `CLFlushRangeInclusive` that takes two pointers and flushes everything at or between them?

Reply:
Should I put it to instr.cc or to another file?

Reply:
`instr.cc` is fine for now. If we accumulate enough "utility, but not architecture specific" functions we can start a new file.

Reply:
I'll create a utils file for more extractions.